OpenSim/Auxiliary/auxiliaryTestFunctions.h

```diff
@@ -28,6 +28,7 @@
 #include <OpenSim/Common/LinearFunction.h>
 #include <OpenSim/Common/PropertyObjArray.h>
 #include "OpenSim/Common/STOFileAdapter.h"
+#include "OpenSim/Simulation/Model/ActivationFiberLengthMuscle.h"
```

Comment:
I was thinking of adding the new function to a new auxiliary header, since this header could be used by tests in the Common library (which should not use headers in downstream libraries).